### ISSUE TITLE
[Catalog] Catalog "Menu" is missing accessibility identifiers for items

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4818]
The catalog (hamburger) menu elements for the 3 items. The items themselves are also inconsistently handled in UIAccessibility. It would be ideal if:

* Each item is an accessibility element
* Each item is includes the trait `UIAccessibilityTraitButton`
* Each item's `accessibilityLabel` is inherited from its UILabel
* Each item's `accessibilityHint` is set/inherited from its UILabel

---
#### Internal data
- Associated internal bug: [b/117176691](http://b/117176691)

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
